modules/sitestudio_config_management/sitestudio_config_management.install

```diff
@@ -19,6 +19,9 @@ function sitestudio_config_management_install($is_syncing) {
     $updated_ignore_config = array_unique(array_merge($existing_ignore_config, $new_ignore_config));
     $config->set('ignored_config_entities', $updated_ignore_config);
     $config->save(TRUE);
+    if (!\Drupal::config('cohesion.sync.settings')->get('enabled_entity_types')) {
```

Comment:
Replace sitestudio_config_management_install with below code
```
if (!$is_syncing) {
    $config_factory = \Drupal::configFactory();
    // Get existing config ignore settings and append the new one.
    $config_ignore = $config_factory->getEditable('config_ignore.settings');
    $existing_ignore_config = $config_ignore->get('ignored_config_entities');
    $new_ignore_config = [
      'cohesion_*',
    ];
    $updated_ignore_config = array_unique(array_merge($existing_ignore_config, $new_ignore_config));
    $config_ignore->set('ignored_config_entities', $updated_ignore_config);
    $config_ignore->save(TRUE);

    if (!$config_factory->getEditable('cohesion.sync.settings')->get('enabled_entity_types')) {
     _sitestudio_config_management_import_configuration('cohesion.sync.settings', 'sitestudio_config_management');
    }
  }
  /** @var \Drupal\sitestudio_config_management\SiteStudioConfigManagement $config_management_service */
  $config_management_service = \Drupal::service("site_studio.config_management");
  $config_management_service->initialize();
```

Reply:
\Drupal::config is an immutableConfig which returns something whereas configFactory would throw error in case config is not present.
 @apathak18